pkg/sidecar/sidecar.go

```diff
@@ -24,20 +25,28 @@ import (
 // Sidecar is the component that consumes the Workload API and renews certs
 // implements the interface Sidecar
 type Sidecar struct {
-	config         *Config
-	client         *workloadapi.Client
-	jwtSource      *workloadapi.JWTSource
-	processRunning int32
-	process        *os.Process
-	certReadyChan  chan struct{}
+	config          *Config
+	client          *workloadapi.Client
+	jwtSource       *workloadapi.JWTSource
+	processRunning  int32
+	process         *os.Process
+	certReadyChan   chan struct{}
+	fileWriteStatus FileWriteStatus
+}
+
+type FileWriteStatus struct {
+	X509WriteSuccess  bool            `json:"x509_write_success"`
+	JwtWriteSuccesses map[string]bool `json:"jwt_write_successes"`
```

Comment:
Should probably make these lower case since they are not accessed directly outside of this package.

Reply:
When I did that, I got a lint error saying the fields needed exported when using JSON tags.
https://github.com/spiffe/spiffe-helper/actions/runs/12794206054/job/35668839624?pr=227

Reply:
Got it, thanks for trying.